magicblock-core/src/link.rs

```diff
@@ -68,6 +73,7 @@ pub fn link() -> (DispatchEndpoints, ValidatorChannelEndpoints) {
         transaction_status: transaction_status_rx,
         account_update: account_update_rx,
         block_update: block_update_rx,
+        tasks_service: Some(tasks_rx),
```

Comment:
nit: Is it necessary to use Option here, the returned container is supposed to be de-structured anyways, so just moving fields out should suffice (as is the case with other receivers).